### PR TITLE
OSM file path in importer, SSH tunnel setup, README.md fix

### PR DESCRIPTION
**Pull request by domupls**
OSM file path is now a required input when importing to database using `import_osm_to_db()`. I also created [`ssh.py`](https://github.com/aicenter/road-graph-tool/blob/importer/python/roadgraphtool/ssh.py) to handle SSH tunnel setup for importing data to database. 
Additionally, I added a few notes to [README.md](https://github.com/aicenter/road-graph-tool/blob/main/README.md) about aforementioned changes.

### CODE REVIEW

File: config-EXAMPLE.ini [hunk 1]
```diff
@@ -6,6 +6,7 @@ server = its.domain.name.com
 server_username = <name>
 
 [database]
+db_type = remote ; or local
```
Comment (marekcuchy) [inline config-EXAMPLE.ini:9]: Instead, you can check if ssh exists in the config. If it is, it is certain that an SSH tunnel needs to be used.

File: python/roadgraphtool/ssh.py
Comment (marekcuchy) [inline python/roadgraphtool/ssh.py:1]: Are you sure that the already implemented tunneling cannot be used?

Comment (marekcuchy) [reply python/roadgraphtool/ssh.py:1]: roadgraphtool.db.set_ssh_to_db_server_and_set_port() does not work?

**Review by marekcuchy — CHANGES_REQUESTED** [review]

File: python/roadgraphtool/ssh.py
Comment (domupls) [reply python/roadgraphtool/ssh.py:1]: It works now, it was poor config setup on my part.

**Comment by domupls** [comment]
Changes have been made

**Review by marekcuchy — APPROVED** [review]

**Pull request merged**